tools/helper/issue_transaction_update.hpp

```diff
@@ -0,0 +1,133 @@
+/*
```

Comment:
all this file looks pretty same, try to use generics and remove terrific code duplication